src/app/project/board/board.component.ts

```diff
@@ -100,13 +112,174 @@ export class BoardComponent implements OnInit {
         this.loading = false;
     }
 
+    getProjectMetadata() {
```

Comment:
This could be already replaced by DSP-JS method call with http, but the payload need to be uploaded first, since there is no default data in the database. Maybe you should request backend to add it.

Reply:
I will add Ajax call to get metadata in next iteration.

Reply:
`HttpClient` from Angular common module.